other-docs/guides/migrating/README.md

```diff
@@ -146,7 +146,6 @@ Next, we need to add some extra configuration to allow Altis to set itself up. O
             "altis/cms-installer": true,
             "altis/dev-tools-command": true,
             "altis/core": true,
-            "altis/local-chassis": true,
```

Comment:
See my comment above.

Reply:
Similar thinking here, this is part of the "migrating to Altis" guide, where we don't really want users to install/use Chassis.